examples/forecast_evaluation/reformat_airnow_rapchemtest.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## MELODIES-MONET-Analysis Airnow prep 
+
+
+import pandas as pd
+import xarray as xr
+import monetio as mio
+from melodies_monet.util import write_util
+
+# Here is an example of how you would use/import a variable from the master script
+# ... python reformat_airnow.py $START_TIME $END_TIME ...
+import sys
+start_time_reformat=sys.argv[1]
+end_time_reformat=sys.argv[2] 
+print(sys.argv[1])
```

Comment:
We could also use the argparse module for command line arguments.  It allows for flexible ordering and optional arguments with defaults.  Here is a typical example of its use:

    import argparse

    parser = argparse.ArgumentParser()
    parser.add_argument('--datadir', type=str,
        default=os.path.join(os.getenv('HOME'), 'Data', 'AirNow'),
        help='top-level data directory (default $HOME/Data/AirNow)')
    parser.add_argument('--debug', action='store_true',
        help='set logging level to debug')
    parser.add_argument('--start', type=str,
        default = datetime.now().strftime('%Y%m%d'),
        help='start date (yyyymmdd)')
    parser.add_argument('--end', type=str,
        default = datetime.now().strftime('%Y%m%d'),
        help='end date (yyyymmdd)')
    args = parser.parse_args()

Reply:
Thanks David. I like this idea. Let's do this for the more generalized versions of these codes.